src/main/resources/com/google/api/codegen/ruby/version_index.snip

```diff
@@ -104,7 +104,11 @@
     AVAILABLE_VERSIONS = Dir["#{FILE_DIR}/*"]
       .select { |file| File.directory?(file) }
       .select { |dir| Google::Gax::VERSION_MATCHER.match(File.basename(dir)) }
-      .select { |dir| File.exist?(File.join(dir, "{@index.postVersionDirPath}.rb")) }
+      @if index.postVersionDirPath
+        .select { |dir| File.exist?(File.join(dir, "{@index.postVersionDirPath}.rb")) }
```

Comment:
Do you think we could add a test for this in the baseline?

Reply:
It already exists in multiple_services.baseline.